rediscluster/nodemanager.py

```diff
@@ -154,10 +154,6 @@ def get_redis_link(self, host, port, decode_responses=False):
     def initialize(self):
         """
         Init the slots cache by asking all startup nodes what the current cluster configuration is
-
```

Comment:
Why remove this comment? It did not feel like you solved it with your new exception catch?

Reply:
I didn't change the behaviour, but it looks like it's already been solved with the break on line 239.

Reply:
Right, i see what you are refering to, great catching this